src/gatling/simulations/uk/gov/hmcts/reform/cmc/performance/simulations/CMCSimulation.scala

```diff
@@ -39,6 +39,19 @@ class CMCSimulation extends Simulation
          .exec(CreateLegalSimulation.createLegalNoClaimScenario)
 
 
+       // NIGHTLY TEST RUNS ///////////
+
+       setUp(
+         scenario1.inject(rampUsers(1) during (200))
```

Comment:
Can we run this for 10 users instead of 1

Reply:
will do 👍

Reply:
done